Speedykom/Regional-Pandemic-Analytics
# fix : display newly created process chain in the process list

## Description
Newly created process chain is not always directly visible in the list and a refresh is need. Repan should check periodically the dags directories and only refresh the process list when the new process is there. 


## Testing Instructions
- Create a new process chain.
- Wait till the Airflow rescans the dags directory.
- The new process chain should be displayed in the process list.
